5-Photoshop/css/style.css

```diff
@@ -0,0 +1,57 @@
+* {
+  margin: 0;
+  background-color: #121116;
+}
+
+.wrapper {
+  background-color: #121116;
+  margin: 120px auto;
+}
+
+.logo {
+  width: 112px;
+  height: 36px;
+  margin: 0 auto;
+}
+
+.weAreSocaial {
+  color: #fefbfc;
+  width: 230px;
+  margin: 50px auto 0;
+  padding: 0 0 22px;
+  text-align: center;
+  border-bottom-style: solid;
+  border-width: 1px;
+  border-color: #fefbfc;
+  font-family: "Open Sans", sans-serif;

```

Comment:
we have the same font on whole page and in that case this style should be added to body and removed from footer